ci.sh

```diff
@@ -66,11 +66,11 @@ fi
 
 python -c "import sys, struct, ssl; print('#' * 70); print('python:', sys.version); print('version_info:', sys.version_info); print('bits:', struct.calcsize('P') * 8); print('openssl:', ssl.OPENSSL_VERSION, ssl.OPENSSL_VERSION_INFO); print('#' * 70)"
 
-python -m pip install -U pip setuptools wheel
+python -m pip install -U pip hatchling build
```

Comment:
Backends are never needed in outer envs — the frontends automatically provision them in the build venvs.

Reply:
```suggestion
python -m pip install -U pip build
```